guardrails/cli/hub/install.py

```diff
@@ -136,7 +136,7 @@ def get_install_url(manifest: ModuleManifest) -> str:
 
 
 def install_hub_module(
-    module_manifest: ModuleManifest, site_packages: str, quiet: bool = False
+    module_manifest: ModuleManifest, site_packages: str, quiet: bool = True
```

Comment:
If we're going to change the default value here, then we should change our `quiet` flag to a `verbose` flag.  The current option doesn't allow specifying an explicit value because it's meant to act as a flag.  This change essentially always forces the `quiet` flag to be enabled.

Reply:
addressed